app/commands/github/team_member/destroy.rb

```diff
@@ -0,0 +1,7 @@
+class Github::TeamMember::Destroy
+  include Mandate
+
+  initialize_with :username, :team
+
+  def call = ::Github::TeamMember.where(username:, team:).delete_all
```

Comment:
```suggestion
  def call = ::Github::TeamMember.where(username:, team:).destroy_all
```

(Safer to use `destroy` for callbacks. But it should definitely match whatever this Command is called)